tests/ui/empty_drop.rs

```diff
@@ -0,0 +1,12 @@
+#![warn(clippy::empty_drop)]
```

Comment:
After using the `span_lint_and_sugg` method, we should test the suggestion with `run-rustfix`

```suggestion
// run-rustfix
#![warn(clippy::empty_drop)]
```